wire-protoc-integration/src/main/java/com/squareup/wire/protocwire/JavaTargetConfig.kt

```diff
@@ -0,0 +1,36 @@
+package com.squareup.wire.protocwire
+
+import com.squareup.wire.schema.JavaTarget
+import com.squareup.wire.gradle.JavaOutput
+
+private const val ANDROID = "android"
+private const val ANDROID_ANNOTATIONS = "androidAnnotations"
+private const val COMPACT = "compact"
+private const val EMIT_DECLARED_OPTIONS = "emitDeclaredOptions"
+private const val EMIT_APPLIED_OPTIONS = "emitAppliedOptions"
+private const val BUILDERS_ONLY = "buildersOnly"
+
+/**
+ * Translates the protoc options into known options for the
+ * JavaTarget via the gradle plugin JavaOutput.
+ */
+internal class JavaTargetConfig {
+  companion object {
+    /**
+     * @param parameters the map of string to string from the protoc options.
```

Comment:
Is there a spec as to how they're defined?

Reply:
I've linked the top comment to https://github.com/buildbreaker/wire/blob/achiu/protoc-plugin/wire-protoc-integration/src/main/java/com/squareup/wire/protocwire/ParameterParser.kt#L21-L28